lib/puppet_x/chocolatey/chocolatey_common.rb

```diff
@@ -62,6 +62,10 @@ def choco_config_file
         chocoInstallPath = PuppetX::Chocolatey::ChocolateyInstall.install_path
         choco_config = "#{chocoInstallPath}\\config\\chocolatey.config"
 
+        # choco may be installed, but a config file doesn't exist until the
+        # first run of choco - trigger that by checking the version
+        choco_run_ensure_config = choco_version
```

Comment:
We typically call `choco -v` during the run and then store the results for the remainder of the run, this just moves that call up earlier.